src/components/subsidy-request-modals/DeclineSubsidyRequestModal.jsx

```diff
@@ -86,6 +88,15 @@ const DeclineSubsidyRequestModal = ({
         >
           Send the learner an email notification
         </Form.Checkbox>
+        <Form.Checkbox
+          className="py-1"
+          data-testid="decline-subsidy-request-modal-unlink-learner-checkbox"
+          checked={shouldUnlinkLearnerFromEnterprise}
+          onChange={(e) => setShouldUnlinkLearnerFromEnterprise(e.target.checked)}
+          description="Your learner won't know they have been disassociated"
```

Comment:
nit: this might a personal preference thing, so feel free to ignore, but should this have a period at the end of the sentence?